composer.json

```diff
@@ -31,7 +31,7 @@
         "grasmash/yaml-cli": "^1.0.0",
         "grasmash/yaml-expander": "^1.2.0",
         "oomphinc/composer-installers-extender": "^1.1",
-        "phpunit/phpunit": "^4.8",
+        "phpunit/phpunit": "^4.8|^5.7|^6.5",
```

Comment:
Let's drop 4.* support.

Reply:
Agreed, it's crazy deprecated, I don't see any reason to keep it, and I've verified that there shouldn't be any conflicts with other dependencies if 5.x is supported.

Reply:
@ba66e77 and I have agreed that we should maintain support for 4.x until we release a new version of BLT. I'll review this PR today.